src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -123,17 +123,11 @@ private static bool IsNamespaceName(NameSyntax name)
             CancellationToken cancellationToken,
             out ClassifiedSpan classifiedSpan)
         {
-            if (symbol != null)
+            // Classify a reference to an attribute constructor in an attribute location
+            // as if we were classifying the attribute type itself.
+            if (symbol.IsConstructor() && name.IsParentKind(SyntaxKind.Attribute))
             {
-                // see through using aliases
-                if (symbol.Kind == SymbolKind.Alias)
```

Comment:
i remove this code because it shoudl never be possible to hit this.  SemanticMOdel never returns aliases when you call GetSymbolInfo.  It only returns aliases if you call GetAliasInfo.  So the need to 'see through' an alias does not arise.